fhirest-core/src/main/java/ee/fhir/fhirest/core/service/conformance/HapiContextHolder.java

```diff
@@ -81,6 +85,9 @@ public class HapiContextHolder implements ConformanceUpdateListener {
   protected FhirValidator validator;
   protected final List<HapiValidationSupportProvider> validationSupportProviders;
 
+  @Value("${CACHE_TIMEOUT:10}")
+  private int minuteCacheTimeout;
```

Comment:
can it be `Duration cacheTimeout` ? spring should support it, if i am not mistaking